http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/AKSInterpreter.scala

```diff
@@ -1026,6 +1055,17 @@ class AKSInterpreter[F[_]](config: AKSInterpreterConfig,
       }
       wsmApi <- wsmClientProvider.getControlledAzureResourceApi(token)
     } yield wsmApi
+
+  // can these two functions be combined somehow? ask a Scala person for the right way to do it
```

Comment:
just caught this stray comment - I'll remove it before merging - but the question stands for any reviewer who wants to answer it!

Reply:
I guess the token-retrieval logic could be DRYed a bit.. as-is seems ok to me though.